resources/asciidoctor/lib/copy_images/extension.rb

```diff
@@ -17,9 +17,23 @@ def initialize name
   end
 
   def process_block block
-    return unless block.context == :image
-    uri = block.image_uri(block.attr 'target')
-    return if Helpers.uriish? uri       # Skip external images
+    if block.context == :image
+      uri = block.image_uri(block.attr 'target')
+      return if Helpers.uriish? uri # Skip external images
+      copy_image block, uri
+    else
+      extension = block.attr 'copy-callout-images'
+      if block.parent && block.parent.context == :colist
```

Comment:
This could be an `elsif` on line 24 right? There's no other in-scope usage of `extension` anyway here.

Reply:
Ah! This is what that empty test block was about! It looks like I never finished it. sorry!